docs/source/1.0/spec/aws/aws-cloudformation.rst

```diff
@@ -640,6 +640,68 @@ The following example defines a CloudFormation resource that has the
         fooAlias: String
     }
 
+.. smithy-trait:: aws.cloudformation#cfnDefaultValue
+.. _aws.cloudformation#cfnDefaultValue-trait:
+
+----------------------------------------
+``aws.cloudformation#cfnDefaultValue`` trait
+----------------------------------------
+
+Summary
+    Indicates that the member annotated has a default value for the resource.
+Trait selector
+    ``resource > operation -[output]-> structure > member``
+
+    *Only applicable to members of ``@output`` operations*
+Value type
+    Annotation trait
+
+Given the following example, because the ``fooAlias``
+member is annotated with ``cfnDefaultValue``, it can be derived
+that the ``fooAlias`` member has a default value for this resource.
```

Comment:
What is the significance of the trait though? I don't understand how it's used.

Reply:
This is mainly for use in drift detection. What this trait does is indicate that the cfn property generated from the member has a default value. For ex if we have this :
```
@output
structure GetFooResponse {
    ....
    @cfnDefaultValue
    timeout: Integer
    ....
 }
```
  
We don't need the actual default value to be passed into the trait in this case. The main behavior we need with this trait is the ability to annotate a member as default so we know the specific members/properties that must match for drift detection.